.github/workflows/notify_slack.yml

```diff
@@ -0,0 +1,39 @@
+name: Test all warehouse platforms
+on:
+  workflow_call:
+    inputs:
+      result:
+        type: string
+        required: true
+      run_id:
+        type: string
+        required: false
+      workflow_name:
```

Comment:
This is not used anywhere, I'm assuming it needs to be in the Slack message.

Reply:
Woops 2